Lib/test/support/__init__.py

```diff
@@ -2118,3 +2119,27 @@ def clear_ignored_deprecations(*tokens: object) -> None:
     if warnings.filters != new_filters:
         warnings.filters[:] = new_filters
         warnings._filters_mutated()
+
+
+# This number should be CPython internal "hot" (C macro QUICKENING_WARMUP_DELAY)
+# count + 1.
+CPYTHON_WARMUP_COUNT = 8+1
```

Comment:
Rather than guessing when the code is "warm", we could add an attribute to the code object. 

So ```Python
     for i in range(CPYTHON_WARMUP_COUNT):
```
becomes:
```
     while not f.__code__._warm:
```

Just be sure not to over-specify `_warm`, and make it clear that how the VM treats "warm" code is very much an implementation detail.